packages/story-editor/src/components/colorPicker/basicColorList.js

```diff
@@ -44,26 +44,66 @@ const focusStyle = css`
     )};
 `;
 
-const SwatchList = styled.div.attrs({
-  role: 'listbox',
-})`
-  display: flex;
-  max-width: 100%;
-  flex-wrap: wrap;
-  gap: 6px;
+const Wrapper = styled.div`
+  display: inline-block;
+`;
+
+const SwatchList = styled.div.attrs({ role: 'listbox' })`
+  width: 100%;
+  display: inline;
+
+  > div {
```

Comment:
Could we use the specific element here, e.g.
`> ${SwatchWrapper}` in this case?

Could all this be assigned to the `SwatchWrapper` directly as well (in case we need to use it)?